docs/development_suite/company/create.mdx

```diff
@@ -7,228 +7,543 @@ sidebar_label: Create a Company
 
-</Tabs>
+      For example, if you project is reachable at the URL **`https://bitbucket.mia-platform.eu/projects/Test`**, then the `basePath` must be **`projects/Test`**.
 
-:::
+    </TabItem>
 
-- `environments` (_required_): an array of objects containing the definitions of the environments for the company. The content of this arrays will be interpolated to replace `%projectId%` by inserting the `projectId` field in the project creation. Each object should contain, for example:
-
-  ```js
-    {
-      "label": "Development",
-      "envId": "development",
-      "envPrefix": "development",
-      "hosts": [
-        {
-          "host": "%projectId%.test.mia-platform.eu",
-          "scheme": "https"
-        },
-        {
-          "host": "cms.%projectId%.test.mia-platform.eu",
-          "isBackoffice": true
-        }
-      ],
-      "cluster": {
-        "clusterId": "human-readable-id-of-the-cluster",
-        "namespace": "%projectId%-development",
-        "kubeContextVariables": {
-          "KUBE_URL": "KUBE_DEV_URL",
-          "KUBE_TOKEN": "KUBE_DEV_TOKEN"
-        }
-      }
-    }
+    </Tabs>
+
+  ##### Example Configuration
+  ```json
+  {
+    "providerId": "my-azure-devops",
+    "type": "azure-devops",
+    "visibility": "private",
+    "basePath": "mia-platform-devops"
+  }
   ```
 
-  :::caution
-  The term _tenantId_ is the new word that refers to the old _companyId_.
+- **pipelines** (_required_):
+  
+  An object that specifies the information regarding the CI/CD pipelines used by the company. 
+
+  :::info
+  This field can also be set by [selecting a default CI/CD tool provider](/docs/development_suite/set-up-infrastructure/providers-management#set-a-default-provider) in the **Providers** section of the Console.
   :::
+  
+  The object must have the following properties:
+
+  - **providerId** (_required_): unique identifier of the CI/CD tool provider associated with the company.
+    
+    It corresponds to the `providerId` field of the provider's CRUD collection and is used to retrieve information about the used provider. 
+
+  - **type** (_required_): one of the supported provider types. Check out the complete list of [supported providers](/docs/development_suite/set-up-infrastructure/providers-management);
 
-  :::warning
-  Do not set in the `cluster.kubeContextVariables` object the plain values to access the cluster. Write the variable key name for the specified environment (as shown in the example). The values saved here are not encrypted!
+  :::note
+  The following properties are supported only when the provider type is **"azure-devops"**:
+
+    - **azurePipelineId**: TODO
+  
   :::
 
-- `pipelines` (_required_): the CI/CD pipelines used by the company, for example:
+  :::note
+  The following properties are supported only when the provider type is **"jenkins"**:
 
+    - **options**: TODO
```

Comment:
This is an object with view and name:

```json
{
  "view": {
    "name": ""
  }
}
```

and will be used during the generation of the service to name the Jenkins pipeline on the correct view